homeassistant/components/simplisafe/__init__.py

```diff
@@ -517,7 +529,7 @@ async def async_update(self):
 
         async def update_system(system):
             """Update a system."""
-            await system.update()
+            await system.update(cached=False)
```

Comment:
We can't do this – their API isn't structured to properly support that. See this attempt for more details: https://github.com/home-assistant/core/pull/30926

Reply:
Why not? See my comment below - it runs fine on SimpliSafe 3.

Reply:
I appreciate the thinking, but again, I don't see the value for door/window sensors that poll for state every 30 seconds; if someone opens/closes a door within that timeframe, HASS will never hear about it.

Reply:
The use case isn't just triggering something when a door/window sensor opens, but it is also to check the current state of the sensor, e.g. check if there are any open windows before leaving the house. There's still value in this use case, and it doesn't require real time sensor data.
Also, we can poll every 15 seconds without any issues.

Reply:
I hear you and can see the validity. Overall, I'm concerned that if tens/hundreds/more of HASS users start pinging SimpliSafe's cloud every 15 seconds in an un-cached way, we're going to get blocked off for good. That's definitely my own fear and not substantiated in any way. 😆 

I'm definitely open to continuing the conversation, but at a minimum, I think the [`simplisafe-python`](https://github.com/bachya/simplisafe-python) work mentioned throughout needs to be looked at before this PR can proceed. Feel free to start looking around and let me know of any questions.